### ISSUE TITLE
OgImage is optional but an og:image tag is undefined when not in use

### ISSUE THREAD

**Issue by skattabrain** [harlan-zw/nuxt-seo-kit#10]
### Details

I got OgImageScreenshot working and it's great, trouble is generating the screenshots is too time and resource consuming so I'm planning to do without `og:image` for now.

If I remove the `<OgImageScreenshot />` tag from my markup, the task doesn't run to generate the images, but it's creating an empty meta tag for `twitter:card` and `og:image`. Wouldn't it be better completely omitted?

```
...
<meta property="twitter:card" content="summary_large_image">
...
<meta property="og:image">
...
```

This is a great plugin BTW, I'm hoping it will continue to be maintained, it's a major time and BS saver!

**Comment by harlan-zw** [comment]
Hey @skattabrain, thanks! And thanks for the issue

I've pushed up a fix for this and it's available in 0.4.14.

I'll be pushing up v1 of nuxt-og-image very soon which adds satori support which should address the time/resource issue. This will be maintained as it's powering a bunch of my sites now :)

**Comment by skattabrain** [comment]
Fantastic! Looking forward to that performance boost!
